### ISSUE TITLE
Add ability to delete playlists

### ISSUE THREAD

**Issue by yagarea** [jmshrv/finamp#543]

**Comment by yagarea** [comment]
> Do you mean the downloaded version on your phone or do you want to delete it from the server?
Deleting playlist from server. Not the songs themself but just playlist.

> The first option already available and it is debatable if the second one is usefull
I find it really useful because absence of this feature makes me dependent on web interface which kills the comfort of an app. I personally think this is basic feature for music management.

**Comment by Nexusnui** [comment]
I am not sure if finamp is supposed to delete files from the server. I also do not know what happens if you delete a playlist, that is based on a m3u list in the web interface.

**Comment by Chaphasilor** [comment]
I agree that this is a fundamental feature that Finamp should support, and in theory it should be as easy as suggested by @yagarea. I just had some bad experiences with deleting playlists in Jellyfin in the past, especially when dealing with .m3u files. I'll have to check if this has gotten more reliable.  
If so, we can probably add this to the beta relatively soon.

**Comment by Chaphasilor** [comment]
This is now possible since beta version 0.9.14 :D  
See #1004 and the [release notes](https://github.com/jmshrv/finamp/releases/tag/0.9.14-beta) for more details!
